src/commands/server/login.ts

```diff
@@ -0,0 +1,152 @@
+/*********************************************************************
+ * Copyright (c) 2020 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+import { Command, flags } from '@oclif/command'
+import { string } from '@oclif/parser/lib/flags'
+import { cli } from 'cli-ux'
+import * as execa from 'execa'
+
+import { CheHelper } from '../../api/che'
+import { CheApiClient } from '../../api/che-api-client'
+import { CheServerLoginManager, LoginRecord } from '../../api/che-login-manager'
+import { KubeHelper } from '../../api/kube'
+import { cheNamespace, CHE_API_ENDPOINT_KEY, username, USERNAME_KEY } from '../../common-flags'
+import { OPENSHIFT_CLI } from '../../util'
+
+const REFRESH_TOKEN_KEY = 'refresh-token'
+const PASSWORD_KEY = 'password'
+export default class Login extends Command {
+  static description = 'Log in to Eclipse Che server'
+
+  static args = [
+    {
+      name: CHE_API_ENDPOINT_KEY,
+      description: 'Eclipse Che server API endpoint',
+      env: 'CHE_API_ENDPOINT',
+      required: false // In case of login via oc token with admin rights
+    }
+  ]
+  static flags: flags.Input<any> = {
+    help: flags.help({ char: 'h' }),
+    chenamespace: cheNamespace,
+    [REFRESH_TOKEN_KEY]: string({
+      char: 't',
+      description: 'Keycloak refresh token',
+      env: 'CHE_KEYCLOAK_REFRESH_TOKEN',
+      required: false,
+    }),
+    [USERNAME_KEY]: username,
+    [PASSWORD_KEY]: string({
+      char: 'p',
+      description: 'Eclipse Che user password',
+      env: 'CHE_USER_PASSWORD',
+      required: false,
+    }),
+  }
+
+  static examples = [
```

Comment:
Add description for every example, it will be useful 
```
  static examples = [
    '# Log in when OpenShift OAuth is not enabled\nserver:login https://che-che.apps-crc.testing/api -u <username> -p <password>\n',
    '# Log in interactively when OpenShift OAuth is not enabled\nserver:login che-che.apps-crc.testing -u <username>\n',
    '# Log in when OpenShift OAuth is enabled\nserver:login che.openshift.io -t <token>\n'
  ]
```
```
  static examples = [
    '# Log in when OpenShift OAuth is not enabled\nserver:login https://che-che.apps-crc.testing/api -u <username> -p <password>\n',
    '# Log in interactively when OpenShift OAuth is not enabled\nserver:login che-che.apps-crc.testing -u <username>\n',
    '# Log in when OpenShift OAuth is enabled\nserver:login che.openshift.io -t <token>\n'
  ]
```